### PR TITLE
PycodeStyleBear:Allow infinite line length

### PR DESCRIPTION
**Pull request by newbazz**
Allow infinite line length when max_line_length
is set to 0.

Closes https://github.com/coala/coala-bears/issues/2265

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by newbazz** [comment]
@Makman2 review ;)

**Comment by Makman2** [comment]
Commit message: Pycode~~S~~**s**tyleBear

File: bears/python/PycodestyleBear.py [hunk 1]
```diff
@@ -37,10 +39,14 @@ def create_arguments(
             these errors are going to be detected.
             See ``pydocstyle`` documentation for a complete list of errors.
         :param max_line_length:
-            Limit lines to this length.
+            Limit lines to this length.Allows infinite line length when
```
Comment (Makman2) [inline bears/python/PycodestyleBear.py]: Missing space after period

File: tests/python/PycodestyleBearTest.py [hunk 2]
```diff
@@ -17,6 +17,9 @@ def hello():
     print("hello world")
 '''
 
+good_file2 = ('def ' + 'h' * 1000 + '():\n' +
```
Comment (Makman2) [inline tests/python/PycodestyleBearTest.py]: hmm calling this file "good" is maybe a bit too much... if possible name it something like `file_with_very_long_line` (you get the idea :+1:) :smile:

**Comment by Makman2** [comment]
unack 0a7cad7e34612327b863d5e1dbb32697dd3c5458

**Comment by newbazz** [comment]
@Makman2 changes made ;)

File: tests/python/PycodestyleBearTest.py [hunk 3]
```diff
@@ -54,3 +57,8 @@ def hello():
     invalid_files=(),
     settings={'max_line_length': 200}
 )
```
Comment (Makman2) [inline tests/python/PycodestyleBearTest.py]: missing blank line

**Comment by Makman2** [comment]
unack 4565d68e1596bfd5c9ad1e63b1be31287b23abe3

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/pull/2269/commits/4dc66492c34e9b0c0abbef3281b18c362a745754

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**